code/modules/power/pt_laser.dm

```diff
@@ -55,13 +56,20 @@
 						terminal = term
 						break dir_loop
 
+		AddComponent(/datum/component/mechanics_holder)
+		SEND_SIGNAL(src,COMSIG_MECHCOMP_ADD_INPUT,"Enable Power Input", PROC_REF(_enable_input_mechchomp))
+		SEND_SIGNAL(src,COMSIG_MECHCOMP_ADD_INPUT,"Disable Power Input", PROC_REF(_disable_input_mechchomp))
```

Comment:
Is there a reason these aren't toggles? Seems like it would be much easier to work with (at least for simple things) and a shorter list of signals is usually less confusing to look at.

Reply:
I was thinking that tracking that toggle would make tracking the state of the PTL lasers annoying, as they shut themselves off if they're not powered. Thinking a little more about that - the PTL output doesn't shut off, it just enters an entirely different state `idle`, so theres really no reason not to leave these as toggle.